### PR TITLE
[Localization] Pledge moves translations

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
None, translation of pending PR #4511 
## Why am I making these changes?
To have it translated

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/ja/arena-tag.json [hunk 1]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (Chapybara-jp) [inline src/locales/ja/arena-tag.json:67]:
```suggestion
  "fireGrassPledgeOnAdd": "場の　周りが　火の海に　包まれた！",
  "fireGrassPledgeOnAddPlayer": "味方の　周りが　火の海に　包まれた！",
  "fireGrassPledgeOnAddEnemy": "相手の　周りが　火の海に　包まれた！",
  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}}は　火の海の　ダメージを受けた！",
  "waterFirePledgeOnAdd": "場の空に　にじが　かかった！",
  "waterFirePledgeOnAddPlayer": "味方の空に　にじが　かかった！",
  "waterFirePledgeOnAddEnemy": "相手の空に　にじが　かかった！",
  "grassWaterPledgeOnAdd": "場の　周りに　湿原が　広がった！",
  "grassWaterPledgeOnAddPlayer": "味方の　周りに　湿原が　広がった！",
  "grassWaterPledgeOnAddEnemy": "相手の　周りに　湿原が　広がった！"
```

File: src/locales/ja/move-trigger.json [hunk 2]
```diff
@@ -68,5 +68,7 @@
   "chillyReception": "{{pokemonName}}は\n寒い　ギャグを　かました！",
   "swapArenaTags": "{{pokemonName}}は\nお互いの　場の　効果を　入れ替えた！",
   "exposedMove": "{{pokemonName}}は {{targetPokemonName}}の\n正体を　見破った！",
-  "afterYou": "{{targetName}}は\nお言葉に　甘えることにした！"
+  "afterYou": "{{targetName}}は\nお言葉に　甘えることにした！",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (Chapybara-jp) [inline src/locales/ja/move-trigger.json:73]:
```suggestion
  "combiningPledge": "２つの技が　１つになった！\nコンビネーション技だ！！",
  "awaitingPledge": "{{userPokemonName}}は　{{allyPokemonName}}を　待っている…"
```

File: src/locales/pt_BR/arena-tag.json [hunk 3]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/arena-tag.json:67]:
```suggestion
  "fireGrassPledgeOnAdd": "Um mar de fogo envolveu o campo!",
  "fireGrassPledgeOnAddPlayer": "Um mar de fogo envolveu sua equipe!",
  "fireGrassPledgeOnAddEnemy": "Um mar de fogo envolveu a equipe adversária!",
  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} foi ferido pelo mar de fogo!",
  "waterFirePledgeOnAdd": "Um arco-íris apareceu no céu!",
  "waterFirePledgeOnAddPlayer": "Um arco-íris apareceu no céu do lado da sua equipe!",
  "waterFirePledgeOnAddEnemy": "Um arco-íris apareceu no céu do lado da equipe adversária!",
  "grassWaterPledgeOnAdd": "Um pântano envolveu o campo!",
  "grassWaterPledgeOnAddPlayer": "Um pântano envolveu sua equipe!",
  "grassWaterPledgeOnAddEnemy": "Um pântano envolveu a equipe adversária!"
```

File: src/locales/pt_BR/move-trigger.json [hunk 4]
```diff
@@ -64,5 +64,7 @@
   "chillyReception": "{{pokemonName}} está prestes a contar uma piada gelada!",
   "exposedMove": "{{pokemonName}} identificou\n{{targetPokemonName}}!",
   "safeguard": "{{targetName}} está protegido por Safeguard!",
-  "afterYou": "{{targetName}} aceitou a gentil oferta!"
+  "afterYou": "{{targetName}} aceitou a gentil oferta!",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (josericardo-fo) [inline src/locales/pt_BR/move-trigger.json:69]:
```suggestion
  "combiningPledge": "Os dois movimentos se juntaram!\nÉ um ataque combinado!",
  "awaitingPledge": "{{userPokemonName}} está esperando pelo movimento de {{allyPokemonName}}..."
```

File: src/locales/ko/arena-tag.json [hunk 5]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (EnochG1) [inline src/locales/ko/arena-tag.json:67]:
```suggestion
  "fireGrassPledgeOnAdd": "필드 주변이\n불바다에 둘러싸였다!",
  "fireGrassPledgeOnAddPlayer": "우리 편 주변이\n불바다에 둘러싸였다!",
  "fireGrassPledgeOnAddEnemy": "상대 주변이\n불바다에 둘러싸였다!",
  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}}[[는]]\n불바다의 데미지를 입었다!",
  "waterFirePledgeOnAdd": "하늘에 무지개가 걸렸다!",
  "waterFirePledgeOnAddPlayer": "우리 편 하늘에 무지개가 걸렸다!",
  "waterFirePledgeOnAddEnemy": "상대 하늘에 무지개가 걸렸다!",
  "grassWaterPledgeOnAdd": "필드에 습지초원이 펼쳐졌다!",
  "grassWaterPledgeOnAddPlayer": "우리 편 주변에\n습지초원이 펼쳐졌다!",
  "grassWaterPledgeOnAddEnemy": "상대 주변에\n습지초원이 펼쳐졌다!"
```

File: src/locales/ko/move-trigger.json [hunk 6]
```diff
@@ -69,5 +69,7 @@
   "chillyReception": "{{pokemonName}}[[는]] 썰렁한 개그를 선보였다!",
   "exposedMove": "{{pokemonName}}[[는]]\n{{targetPokemonName}}의 정체를 꿰뚫어 보았다!",
   "safeguard": "{{targetName}}[[는]] 신비의 베일이 지켜 주고 있다!",
-  "afterYou": "{{targetName}}[[는]]\n배려를 받아들이기로 했다!"
+  "afterYou": "{{targetName}}[[는]]\n배려를 받아들이기로 했다!",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (EnochG1) [inline src/locales/ko/move-trigger.json:74]:
```suggestion
  "combiningPledge": "2개의 기술이 하나가 되었다!\n콤비네이션 기술이다!",
  "awaitingPledge": "{{userPokemonName}}[[는]]\n{{allyPokemonName}}[[를]] 기다리고 있다..."
```

File: src/locales/de/arena-tag.json [hunk 7]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (CodeTappert) [inline src/locales/de/arena-tag.json:67]:
```suggestion
  "fireGrassPledgeOnAddPlayer": "Um die Pokémon auf deiner Seite erstreckt sich ein Meer aus Feuer!",
  "fireGrassPledgeOnAddEnemy": "Um die Pokémon auf der gegnerischen Seite erstreckt sich ein Meer aus Feuer!",
  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} nimmt Schaden durch das Meer aus Feuer!",
  "waterFirePledgeOnAdd": "Ein Regenbogen erscheint am Himmel!",
  "waterFirePledgeOnAddPlayer": "Ein Regenbogen erscheint am Himmel über den Pokémon auf deiner Seite!",
  "waterFirePledgeOnAddEnemy": "Ein Regenbogen erscheint am Himmel über den Pokémon auf der gegnerischen Seite!",
  "grassWaterPledgeOnAdd": "Ein Sumpf tut sich auf!",
  "grassWaterPledgeOnAddPlayer": "Ein Sumpf tut sich um die Pokémon auf deiner Seite auf!",
  "grassWaterPledgeOnAddEnemy": "Ein Sumpf tut sich um die Pokémon auf der gegnerischen Seite auf!"
```

File: src/locales/de/move-trigger.json [hunk 8]
```diff
@@ -71,5 +71,7 @@
   "safeguard": "{{targetName}} wird durch Bodyguard geschützt!",
   "substituteOnOverlap": "{{pokemonName}} hat bereits einen Delegator!",
   "substituteNotEnoughHp": "Es ist zu schwach, um einen Delegator einzusetzen!",
-  "afterYou": "{{targetName}} lässt sich auf Galanterie ein!"
+  "afterYou": "{{targetName}} lässt sich auf Galanterie ein!",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
```
Comment (CodeTappert) [inline src/locales/de/move-trigger.json:75]:
```suggestion
  "combiningPledge": "Zwei Attacken bilden zusammen eine Kombi-Attacke!",
```

File: src/locales/de/move-trigger.json [hunk 9]
```diff
@@ -71,5 +71,7 @@
   "safeguard": "{{targetName}} wird durch Bodyguard geschützt!",
   "substituteOnOverlap": "{{pokemonName}} hat bereits einen Delegator!",
   "substituteNotEnoughHp": "Es ist zu schwach, um einen Delegator einzusetzen!",
-  "afterYou": "{{targetName}} lässt sich auf Galanterie ein!"
+  "afterYou": "{{targetName}} lässt sich auf Galanterie ein!",
+  "combiningPledge": "Zwei Attacken bilden zusammen eine Kombi-Attacke!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (CodeTappert) [inline src/locales/de/move-trigger.json:76]:
```suggestion
  "awaitingPledge": "{{userPokemonName}} wartet auf {{allyPokemonName}}..."
```

File: src/locales/zh_TW/arena-tag.json [hunk 10]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (40chyan) [inline src/locales/zh_TW/arena-tag.json:67]:
```suggestion
"fireGrassPledgeOnAdd": "场地被火海包围了!",
"fireGrassPledgeOnAddPlayer": "我方周围被火海包围了!",
"fireGrassPledgeOnAddEnemy": "对手周围被火海包围了!",
"fireGrassPledgeLapse": "{{pokemonNameWithAffix}}受到了火海的伤害!",
"waterFirePledgeOnAdd": "彩虹出现在了上空!",
"waterFirePledgeOnAddPlayer": "彩虹出现在了我方上空!",
"waterFirePledgeOnAddEnemy": "彩虹出现在了对手上空!",
"grassWaterPledgeOnAdd": "场地周围延伸出了湿地!",
"grassWaterPledgeOnAddPlayer": "在我方周围延伸出了湿地!",
"grassWaterPledgeOnAddEnemy": "在对手周围延伸出了湿地!"
```

**Review by 40chyan — COMMENTED** [review]
zhcn

File: src/locales/zh_CN/arena-tag.json [hunk 11]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (40chyan) [inline src/locales/zh_CN/arena-tag.json:67]:
```suggestion
"fireGrassPledgeOnAdd": "场地被火海包围了！",
"fireGrassPledgeOnAddPlayer": "我方周围被火海包围了！",
"fireGrassPledgeOnAddEnemy": "对手周围被火海包围了！",
"fireGrassPledgeLapse": "{{pokemonNameWithAffix}}受到了火海的伤害！",
"waterFirePledgeOnAdd": "彩虹出现在了上空！",
"waterFirePledgeOnAddPlayer": "彩虹出现在了我方上空！",
"waterFirePledgeOnAddEnemy": "彩虹出现在了对手上空！",
"grassWaterPledgeOnAdd": "场地周围延伸出了湿地！",
"grassWaterPledgeOnAddPlayer": "在我方周围延伸出了湿地！",
"grassWaterPledgeOnAddEnemy": "在对手周围延伸出了湿地！"
```

File: src/locales/zh_CN/move-trigger.json [hunk 12]
```diff
@@ -68,5 +68,7 @@
   "chillyReception": "{{pokemonName}}\n说出了冷笑话！",
   "exposedMove": "{{pokemonName}}识破了\n{{targetPokemonName}}的原型！",
   "safeguard": "{{targetName}}\n正受到神秘之幕的保护！",
-  "afterYou": "{{targetName}}\n接受了对手的好意！"
+  "afterYou": "{{targetName}}\n接受了对手的好意！",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (40chyan) [inline src/locales/zh_CN/move-trigger.json:73]:
```suggestion
  "combiningPledge": "两个招式合二为一！\n这是合体招式！！",
  "awaitingPledge": "{{userPokemonName}}正在等待\n{{allyPokemonName}}的招式"
```

File: src/locales/es/arena-tag.json [hunk 13]
```diff
@@ -54,5 +54,15 @@
...
+  "fireGrassPledgeOnAdd": "A sea of fire enveloped the field!",
+  "fireGrassPledgeOnAddPlayer": "A sea of fire enveloped your team!",
+  "fireGrassPledgeOnAddEnemy": "A sea of fire enveloped the opposing team!",
+  "fireGrassPledgeLapse": "{{pokemonNameWithAffix}} was hurt by the sea of fire!",
+  "waterFirePledgeOnAdd": "A rainbow appeared in the sky!",
+  "waterFirePledgeOnAddPlayer": "A rainbow appeared in the sky on your team's side!",
+  "waterFirePledgeOnAddEnemy": "A rainbow appeared in the sky on the opposing team's side!",
+  "grassWaterPledgeOnAdd": "A swamp enveloped the field!",
+  "grassWaterPledgeOnAddPlayer": "A swamp enveloped your team!",
+  "grassWaterPledgeOnAddEnemy": "A swamp enveloped the opposing team!"
```
Comment (DanStevensonx) [inline src/locales/es/arena-tag.json:67]:
```suggestion
  "safeguardOnRemoveEnemy": "El efecto de Velo Sagrado en el equipo enemigo se ha disipado.",
  "fireGrassPledgeOnAdd": "¡El terreno fué envuelto en un mar de llamas!",
  "fireGrassPledgeOnAddPlayer": "¡Tu equipo se ve rodeado por un mar de llamas!",
  "fireGrassPledgeOnAddEnemy": "¡El equipo rival se ve rodeado por un mar de llamas!",
  "fireGrassPledgeLapse": "¡{{pokemonNameWithAffix}} ha resultado herido por un mar de llamas!",
  "waterFirePledgeOnAdd": "¡Ha aparecido un arcoíris en el cielo!",
  "waterFirePledgeOnAddPlayer": "¡Ha aparecido un arcoíris sobre tu equipo!",
  "waterFirePledgeOnAddEnemy": "¡Ha aparecido un arcoíris sobre el equipo rival!",
  "grassWaterPledgeOnAdd": "¡El terreno fué envuelto en un pantano!",
  "grassWaterPledgeOnAddPlayer": "¡Ha aparecido un pantano alrededor de tu equipo!",
  "grassWaterPledgeOnAddEnemy": "¡Ha aparecido un pantano alrededor del equipo rival!"
```

File: src/locales/es/move-trigger.json [hunk 14]
```diff
@@ -67,5 +67,7 @@
   "swapArenaTags": "¡{{pokemonName}} ha intercambiado los efectos del terreno de combate!",
   "exposedMove": "¡{{pokemonName}} ha identificado\n{{targetPokemonName}}!",
   "safeguard": "¡{{targetName}} está protegido por Velo Sagrado!",
-  "afterYou": "¡{{targetName}} ha decidido aprovechar la oportunidad!"
+  "afterYou": "¡{{targetName}} ha decidido aprovechar la oportunidad!",
+  "combiningPledge": "The two moves have become one!\nIt's a combined move!",
+  "awaitingPledge": "{{userPokemonName}} is waiting for {{allyPokemonName}}'s move..."
```
Comment (DanStevensonx) [inline src/locales/es/move-trigger.json:72]:
```suggestion
  "afterYou": "¡{{targetName}} ha decidido aprovechar la oportunidad!",
  "combiningPledge": "¡Los dos movimientos se han unido!\n¡Es un movimiento combinado!",
  "awaitingPledge": "El {{userPokemonName}} está esperando al movimiento de {{allyPokemonName}}..."
```

**Review by DanStevensonx — COMMENTED** [review]
Translated to Spanish

**Review by Chapybara-jp — APPROVED** [review]

**Pull request closed without merging**